README.md

```diff
@@ -5,7 +5,7 @@ A custom element that shows text as if it were being typed
 ## Installation
 
 ```
-$ npm install @github/typing-effect-element
+npm install @github/typing-effect-element
```

Comment:
Why remove this?

Reply:
I think this might have been a markdown linter change, I'll revert this change.